docs/adding games.md

```diff
@@ -94,39 +95,10 @@ The first step is to research your game. Even if you've been dealt the worst han
 it's possible other motivated parties have concocted useful tools for your game already. 
 Always be sure to search the Internet for the efforts of other modders.  
   
-### Analysis Tools
-Depending on the game’s underlying engine, there may be some tools you can use either in lieu of or in addition to existing game tools.  
-  
-#### [dnSpy](https://github.com/dnSpy/dnSpy/releases)  
-The first tool in your toolbox is dnSpy. 
-dnSpy is useful for opening and modifying code files, like .exe and .dll files, that were made in C#. 
-This won't work for executable files made by other means, and obfuscated code (code which was deliberately made 
-difficult to reverse engineer) will thwart it, but 9 times out of 10 this is exactly what you need. 
-You'll want to avoid opening common library files in dnSpy, as these are unlikely to contain the data you're looking to 
-modify. 
+### Other helpful tools
+Depending on the game’s underlying engine, there may be some tools you can use either in lieu of or in addition to
+existing game tools.
 
-For Unity games, the file you’ll want to open will be the file (Data Folder)/Managed/Assembly-CSharp.dll, as pictured below:  
-  
-![Heavy Bullets Managed Directory in Window's Explorer](./img/heavy-bullets-managed-directory.png)  
```

Comment:
The image file should probably deleted as part of the PR. It can still be dug up in the history if need be.